.github/workflows/build.yml

```diff
@@ -0,0 +1,35 @@
+name: Debug Build
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+      - develop
+# Where will they run
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+        submodules: recursive
+    - uses: actions/setup-java@v2
+      with:
+        java-version: 11
+        distribution: 'temurin'
+        cache: 'gradle'
+    - name: Build org.tiqr.authenticator APK
+      run: bash ./gradlew assembleDebug --stacktrace
+    - name: Build org.tiqr.authenticator.testing APK
+      run: bash ./gradlew assembleAcceptance --stacktrace
+    - name: Upload APK
+      uses: actions/upload-artifact@v1
+      with:
+        name: production-debug
+        path: app/build/outputs/apk/production/debug/app-production-debug.apk
```

Comment:
Debug versions are meant for local testing only. When doing actual QA testing, use the release versions instead of the debug one.

Reply:
My intention is to create intermediate apk's for local testing. QA tests will be done using tagged releases, generated by release.yml

Reply:
Even in that case, use the `release` suffix. Debug is only mean to use from within Android Studio

Reply:
The package installer won't install it anyway, because debug apks are not installable from the file system, only via `adb`.

Reply:
changed to production release